classes/class-ppc-pagesetups.php

```diff
@@ -174,7 +174,12 @@ public function ppc_custom_box_html() {
 					?>
 					<label for="<?php echo esc_attr( $ppc_key ); ?>">
 					<div class="ppc-checklist-item-wrapper">
-						<input type="checkbox" name="checkbox[]" id="<?php echo esc_attr( $ppc_key ); ?>" class="ppc_checkboxes" value= "<?php echo esc_attr( $ppc_key ); ?>" <?php checked( true, array_key_exists( $ppc_key, $value ) ); ?> >
+						<input type="checkbox" name="checkbox[]" id="<?php echo esc_attr( $ppc_key ); ?>" class="ppc_checkboxes" value= "<?php echo esc_attr( $ppc_key ); ?>" 
+																				<?php
+																				if ( ! empty( $value ) ) {
+																					checked( true, array_key_exists( $ppc_key, $value ) ); }
+																				?>
```

Comment:
@shubhw12 You can use ternary condition here instead of if condition in between the HTML markup.

`empty( $value ) ) ? '' : checked( true, array_key_exists( $ppc_key, $value ) );`

Reply:
Done with these changes.